elftools/dwarf/enums.py

```diff
@@ -251,35 +260,55 @@
 
 
 ENUM_DW_FORM = dict(
-    DW_FORM_null            = 0x00,
-    DW_FORM_addr            = 0x01,
-    DW_FORM_block2          = 0x03,
-    DW_FORM_block4          = 0x04,
-    DW_FORM_data2           = 0x05,
-    DW_FORM_data4           = 0x06,
-    DW_FORM_data8           = 0x07,
-    DW_FORM_string          = 0x08,
-    DW_FORM_block           = 0x09,
-    DW_FORM_block1          = 0x0a,
-    DW_FORM_data1           = 0x0b,
-    DW_FORM_flag            = 0x0c,
-    DW_FORM_sdata           = 0x0d,
-    DW_FORM_strp            = 0x0e,
-    DW_FORM_udata           = 0x0f,
-    DW_FORM_ref_addr        = 0x10,
-    DW_FORM_ref1            = 0x11,
-    DW_FORM_ref2            = 0x12,
-    DW_FORM_ref4            = 0x13,
-    DW_FORM_ref8            = 0x14,
-    DW_FORM_ref_udata       = 0x15,
-    DW_FORM_indirect        = 0x16,
-    DW_FORM_sec_offset      = 0x17,
-    DW_FORM_exprloc         = 0x18,
-    DW_FORM_flag_present    = 0x19,
-    DW_FORM_ref_sig8        = 0x20,
+    DW_FORM_null                = 0x00,
+    DW_FORM_addr                = 0x01,
+    DW_FORM_block2              = 0x03,
+    DW_FORM_block4              = 0x04,
+    DW_FORM_data2               = 0x05,
+    DW_FORM_data4               = 0x06,
+    DW_FORM_data8               = 0x07,
+    DW_FORM_string              = 0x08,
+    DW_FORM_block               = 0x09,
+    DW_FORM_block1              = 0x0a,
+    DW_FORM_data1               = 0x0b,
+    DW_FORM_flag                = 0x0c,
+    DW_FORM_sdata               = 0x0d,
+    DW_FORM_strp                = 0x0e,
+    DW_FORM_udata               = 0x0f,
+    DW_FORM_ref_addr            = 0x10,
+    DW_FORM_ref1                = 0x11,
+    DW_FORM_ref2                = 0x12,
+    DW_FORM_ref4                = 0x13,
+    DW_FORM_ref8                = 0x14,
+    DW_FORM_ref_udata           = 0x15,
+    DW_FORM_indirect            = 0x16,
+    DW_FORM_sec_offset          = 0x17,
+    DW_FORM_exprloc             = 0x18,
+    DW_FORM_flag_present        = 0x19,
+    DW_FORM_strx                = 0x1a,
+    DW_FORM_addrx               = 0x1b,
+    DW_FORM_ref_sup4            = 0x1c,
+    DW_FORM_strp_sup            = 0x1d,
+    DW_FORM_data16              = 0x1e,
+    DW_FORM_line_strp           = 0x1f,
+    DW_FORM_ref_sig8            = 0x20,
+    DW_FORM_implicit_const      = 0x21,
+    DW_FORM_loclistx            = 0x22,
+    DW_FORM_rnglistx            = 0x23,
+    DW_FORM_ref_sup8            = 0x24,
+    DW_FORM_strx1               = 0x25,
+    DW_FORM_strx2               = 0x26,
+    DW_FORM_strx3               = 0x27,
+    DW_FORM_strx4               = 0x28,
+    DW_FORM_addrx1              = 0x29,
+    DW_FORM_addrx2              = 0x2a,
+    DW_FORM_addrx3              = 0x2b,
+    DW_FORM_addrx4              = 0x2c,
 
-    DW_FORM_GNU_strp_alt    = 0x1f21,
+    DW_FORM_GNU_addr_index  = 0x1f01,
```

Comment:
Again, can you please just double check the formatting here? Seems like indentation changed by from line 308 the `=` is at a different column

Reply:
Yeah, that was intentional in this case -- I noticed that some of the groupings were only locally aligned, not aligned as a whole. For 308 onwards I just made the new `DW_FORM_GNU` enums align with the existing ones, and didn't make them align with the non-vendor-specific enums. I can change that, though.

Reply:
Thanks for checking. I'll just merge it and fix it up if it looks off